src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,125 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static int initialCapacity = 16;
+    private static double initialLoadFactor = 0.75;
+    private Entry<K, V>[] table = new Entry[initialCapacity];
```

Comment:
You should parameterize the `Entry<K, V>[]` array. You can read about it [here](https://www.baeldung.com/java-diamond-operator).

Reply:
It is!